_locales/en/messages.json

```diff
@@ -2346,7 +2346,7 @@
 			"days": {
 				"content": "$1",
 				"example": "info@ghostery.com"
```

Comment:
The `example: info@ghostery.com` is not an accurate example.  Remove it, or make a more realistic example.
Also, do you need to do anything with `day` vs `days` for when `number-of-days == 1`?